src/main/webapp/app/config/constants.tsx

```diff
@@ -104,7 +104,7 @@ export const SHORTEN_TEXT_FROM_LIST_THRESHOLD = 5;
 export const TOKEN_ABOUT_2_EXPIRE_NOTICE_IN_DAYS = 14;
 export const USAGE_TOP_USERS_LIMIT = 10000;
 export const USAGE_ALL_TIME_KEY = 'All';
-export const USAGE_ALL_TIME_VALUE = 'This year';
+export const USAGE_ALL_TIME_VALUE = 'Past Year';
```

Comment:
<img width="1265" alt="Screenshot 2024-03-20 at 4 35 41 PM" src="https://github.com/oncokb/oncokb-public/assets/71607240/d505c681-65dd-4002-bb3c-e80f50384ef2">

It does the past year, i.e. **one year back from today**. Is this confusing/not what we wanted?

Reply:
@benzuzu oh I see. `Last 12 Months` is definitely better.

Reply:
@zhx828 Should we also change `By Year` tab on the top to 
- `All Time (Last 12 Months)`
- `Last 12 Months`
- `All Time`
- `Cumulative (Last 12 Months)`
or does `By Year` make sense?

Reply:
@benzuzu I'm a bit confused. Looking at the stats for the user, it's past 12 month. But for resource, it's from the starting of the year to current?

Reply:
@zhx828 ah, sorry. I was wrong - it's been a bit since I wrote the logic. Both the user and resource stats are beginning of year to current (not last 12 months). If I remember correctly, we decided to change backed logic in `userOverviewUsageGet` specifically so that we could view specific user usage for the past year (not just year to date). I've changed the tab now below:

<img width="1256" alt="Screenshot 2024-03-20 at 5 52 15 PM" src="https://github.com/oncokb/oncokb-public/assets/71607240/7ac84344-aa24-4ff2-82b0-38a10c3c389f">
<img width="1263" alt="Screenshot 2024-03-20 at 5 52 22 PM" src="https://github.com/oncokb/oncokb-public/assets/71607240/e29aa49d-c684-4959-9f5e-234e2342401b">

I can change the backend logic for `resourceUsageGet` as well to add all 12 months, but I'm not sure if it is worth the trouble.